res/convert_to_tutorial.py

```
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Ruff (0.15.10)

[warning] Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)